packages/electron-redux-e2e/package.json

```diff
@@ -9,7 +9,7 @@
   "private": true,
   "dependencies": {
     "electron": "^1.7.9",
-    "electron-redux": "^1.3.1",
+    "electron-redux": "^1.3.3-0",
```

Comment:
This should be `^1.4.0` if we bump to minor. It's also why the build is currently failing: https://circleci.com/gh/hardchor/electron-redux/421

Reply:
I'm living the versions untouched. Lerna would take care of it, I suppose.